theoplayer/getting-started/01-sdks/02-android/00-getting-started.mdx

```diff
@@ -0,0 +1,476 @@
+                google()
+                mavenCentral()
+                maven { url = uri("https://maven.theoplayer.com/releases") }
+            }
+        }
+        ```
+        </TabItem>
+        <!-- prettier-ignore-end -->
+    </Tabs>
+
+2. Next, we need to edit the `build.gradle` on the **module-level** (**app/build.gradle**)
+
+
+    <Tabs queryString="lang">
+       <!-- prettier-ignore-start -->
+       <TabItem value="groovy" label="Groovy">
+        ```groovy
+        implementation "com.theoplayer.theoplayer-sdk-android:core:+"
+        implementation "org.jetbrains.kotlin:kotlin-stdlib:1.6.21"
+        implementation "androidx.appcompat:appcompat:1.7.0"
+        ```
+        </TabItem>
+        <TabItem value="kotlin" label="Kotlin">
+        ```kotlin
+        implementation("com.theoplayer.theoplayer-sdk-android:core:+")
+        implementation("org.jetbrains.kotlin:kotlin-stdlib:1.6.21")
+        implementation("androidx.appcompat:appcompat:1.7.0")
+        ```
+        </TabItem>
+        <!-- prettier-ignore-end -->
+    </Tabs>
+
+<u>Note</u>: It's possible to replace the `+` with a particular version of THEOplayer that should be used, for example:
+
+`implementation 'com.theoplayer.theoplayer-sdk-android:core:8.0.0'`
+
+## Setting up the license
+
+You need to configure your THEOplayer license, if you don't enter this license string, you might encounter licensing issues (e.g.: when trying to play non-whitelisted video streams).
+
+First, obtain a THEOplayer Android SDK license through the [THEOplayer Developer Portal](https://portal.theoplayer.com/login).
+![](https://cdn.theoplayer.com/images/git/theoplayer-android-sdk-license-string.png)
+
+Then, identify your license string from the screenshot above and add the `<meta-data ...` tag below at the appropriate location in your `AndroidManifest.xml`, and swap `your_license_here` with your
+actual license string.
+
+```xml
+<application>
+    <meta-data
+        android:name="THEOPLAYER_LICENSE"
+        android:value="your_license_here" />
+</application>
+```
+
+Additionally, to play online videos with THEOplayer, you first need to add [INTERNET](https://developer.android.com/reference/android/Manifest.permission.html#INTERNET) permission into the `AndroidManifest.xml` file too.
+
+```xml
+<uses-permission android:name="android.permission.INTERNET" />
+```
+
+More information regarding the license can be found [here](../../../how-to-guides/12-license/00-introduction.md).
+
+## Add THEOplayer view to your application
+
+THEOplayer can be added to an application either through a layout (i.e. XML) or programmatically through the constructor API. This guide uses the layout approach. Refer to [Using the constructor API](https://github.com/THEOplayer/theoplayer-sdk-android#b-using-the-constructor-api)
+if you want to use the latter approach. This approach also allows you to configure the license string through a programmatic configuration instead of the `AndroidManifest.xml`.
```

Comment:
We should move the "Using the constructor API" section into this article too.

Reply:
Done